README.md

```diff
@@ -24,18 +24,34 @@ Install Node modules:
 npm install
 ```
 
-Install the docker container with the database:
+Start the docker containers:
 ```
-docker run --name rra-postgis \
--e "POSTGRES_PASSWORD=rra" \
--e "POSTGRES_USER=rra" \
--e "POSTGRES_DB=rra" \
--p 5432:5432 \
--d \
-mdillon/postgis
+docker-compose up
```

Comment:
I like the output for development purposes, but it's better to default to background.